src/components/TodoFilter/TodoFilter.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { FILTER } from '../../constants';
+
+export const TodoFilter = ({ todosType, selectTodosType }) => (
+  <ul className="filters">
+    <li>
+      <a
+        href="#/"
+        className={classNames({
+          selected: todosType === FILTER.all,
+        })}
+        onClick={() => selectTodosType(FILTER.all)}
+      >
+        {FILTER.all}
+      </a>
+    </li>
+
+    <li>
+      <a
+        href="#/active"
```

Comment:
you can also create an array with all necessary data about the buttons and map it, this will allow you to write code for them once

Reply:
done